langs/python.go

```diff
@@ -64,6 +67,85 @@ func (h *PythonLangHelper) IsMultiStage() bool {
 	return false
 }
 
+// HasBoilerplate return whether the Python runtime has boilerplate that can be generated
+func (h *PythonLangHelper) HasBoilerplate() bool { return true }
+
+// GenerateBoilerplate creates the func file and test file for the Python runtime
+func (h *PythonLangHelper) GenerateBoilerplate() error {
+	wd, err := os.Getwd()
+	if err != nil {
+		return err
+	}
+	codeFile := filepath.Join(wd, "func.py")
+	if exists(codeFile) {
+		return ErrBoilerplateExists
+	}
+	testFile := filepath.Join(wd, "test.json")
+	if exists(testFile) {
+		return ErrBoilerplateExists
+	}
+
+	if err := ioutil.WriteFile(codeFile, []byte(helloPythonSrcBoilerplate), os.FileMode(0644)); err != nil {
+		return err
+	}
+
+	if err := ioutil.WriteFile(testFile, []byte(pythonTestBoilerPlate), os.FileMode(0644)); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+const (
+	helloPythonSrcBoilerplate = `
+import sys
```

Comment:
This boilerplate is wrong for anything except default format. This type of the boilerplate works only for one particular case - non-fdk default format. The boilerplate should be generated with respect to https://github.com/fnproject/fdk-python. But that is not all, fdk-python would never support python2, so if the python runtime version is 2.x CLI should fail.